pandas/tests/indexes/test_base.py

```diff
@@ -591,12 +591,19 @@ def test_empty_fancy(self):
             # Index.
             pytest.raises(IndexError, idx.__getitem__, empty_farr)
 
-    def test_getitem(self):
```

Comment:
this original test is pretty useless, so no need to keep its function (and this is tested else in many places as well)
all of the indexes have a ``test_indexing.py``

Reply:
just rename the test to ``test_getitem_error``. if you want you can split this out into a new file, ``test_indexing.py`` in this dir.